src/api/rest/server/server.go

```diff
@@ -67,9 +67,11 @@ func ApiServer() {
 
 	e.GET("/swagger/*", echoSwagger.WrapHandler)
 
-	e.GET("/", func(c echo.Context) error {
-		return c.String(http.StatusOK, "Hello, World! This is cloud-barista cb-tumblebug")
-	})
+	e.GET("/", rest_common.Healthy)
+	e.GET("/healthy", rest_common.Healthy)
+	e.GET("/tumblebug", rest_common.Healthy)
+	e.GET("/tumblebug/healthy", rest_common.Healthy)
```

Comment:
컨벤션들을 좀 찾아봤는데, health 로 많이 사용하고 있네요.. ^^
https://goinbigdata.com/how-to-create-health-check-for-restful-microservice-in-golang/

아래와 같이 처리하면 어떨까요?
healthy -> health (Get 건강한 -> Get 건강상태)
tumblebug path는 prefix로 사용되므로 포함 (api 게이트웨이에서 사용 필요..?)

```suggestion
	e.GET("/tumblebug/health", rest_common.Health)
```

Reply:
의견 감사합니다. 반영했습니다!